python/lsst/ts/scheduler/scheduler_csc.py

```diff
@@ -1045,6 +1226,15 @@ async def check_script_info(self, data: salobj.BaseDdsDataType) -> None:
         """
         await self.model.callback_script_info(data=data)
 
+        if (
+            "lock_target_loop_and_check_targets" not in self._tasks
+            or self._tasks["lock_target_loop_and_check_targets"].done()
+        ):
+            self._tasks["lock_target_loop_and_check_targets"] = asyncio.create_task(
```

Comment:
That's three times the string literal `lock_target_loop_and_check_targets` in the past four lines. Please extract to a constant and use the constant here.